server/metrics/metrics.go

```diff
@@ -354,6 +353,7 @@ func NewMetrics(info InstanceInfo) Metrics {
 	}, []string{"action"})
 	m.registry.MustRegister(m.syncMsgFileDelayTime)
 
+	// connected, invited, whitelisted
```

Comment:
Does this mean `connected + invited + whitelisted`? I'd like to make sure we can still measure actual connections separately.

Reply:
No, that's just a section header (ending on `:403`). Each metric is still separate.